flake.nix

```diff
@@ -61,6 +61,7 @@
       lenovo-thinkpad-x1-6th-gen = import ./lenovo/thinkpad/x1/6th-gen;
       lenovo-thinkpad-x1-7th-gen = import ./lenovo/thinkpad/x1/7th-gen;
       lenovo-thinkpad-x1-extreme = import ./lenovo/thinkpad/x1-extreme;
+      microsoft-surface = import ./microsoft/surface;
```

Comment:
On related notes should surface-pro/3/default.nix import microsoft-surface?

Reply:
Since the original PR adding the new configuration didn’t remove the Pro 3 configuration, I kept it like it was.

That should work though.

Reply:
@mexisme should we import the more generic module also for this module?

Reply:
Per [the wiki](https://github.com/linux-surface/linux-surface/wiki/Supported-Devices-and-Features), SP3 is supported by the `linux-surface` patches, so it should be safe.

Reply:
Sorry for taking this PR hostage. If this is the case, we should just do it now because otherwise it will never happen.

Reply:
> @mexisme should we import the more generic module also for this module?

In general, I would say so; the intent of the linux-surface patches is to support all Surface devices.

But I've just woken, and can't fully remember the content of the Surface 3 specific derivation.

Reply:
> and can't fully remember the content of the Surface 3 specific derivation.

@mexisme It’s a single option that’s definitely made obsolete with the custom kernel, and doesn’t even include the custom kernel.
Based on this, I think it’s safe to completely remove.